content/build/building/base-images.md

```diff
@@ -9,58 +9,66 @@ aliases:
 - /develop/develop-images/baseimages/
 ---
 
-Most Dockerfiles start from a parent image. If you need to completely control
-the contents of your image, you might need to create a base image instead.
-Here's the difference:
+All Dockerfiles start from a base image. A base is the image that your image is based on.
+It refers to the contents of the `FROM` instruction in the Dockerfile.
 
-- A [parent image](../../glossary.md#parent-image) is the image that your
-  image is based on. It refers to the contents of the `FROM` directive in the
-  Dockerfile. Each subsequent declaration in the Dockerfile modifies this parent
-  image. Most Dockerfiles start from a parent image, rather than a base image.
-  However, the terms are sometimes used interchangeably.
+The following Dockerfile example uses the `python` Docker Official Image as a base image.
+Each subsequent instruction in the Dockerfile modifies the base image, adding or removing files.
```

Comment:
```suggestion
Each subsequent instruction in the Dockerfile extends the base image, adding or removing files.
```

Don't want to give the impression that you're actually modifying the base image itself.

Reply:
Perhaps we should avoid avoid "or removing"; it's not common to remove files from a base image, and mentioning it here could quickly go down the rabbit-hole of describing layers, and how files cannot be removed from previous layers etc etc.

Reply:
Thanks, I've fixed both of these